pathpy/HigherOrderNetwork.py

```diff
@@ -873,31 +916,48 @@ def getTransitionMatrix(self, includeSubPaths=True):
         @param includeSubpaths: whether or not to include subpath statistics in the 
             transition probability calculation (default True)
         """
-        row = []
-        col = []
-        data = []
         D = self.degrees(includeSubPaths=includeSubPaths, weighted=True, mode='OUT')
-        for (s,t) in self.edges:
-            # either s->t has been observed as a longest path, or we are interested in subpaths as well
-
-            # the following makes sure that we do not accidentially consider zero-weight edges (automatically added by default_dic)
-            if (self.edges[(s,t)][1] > 0) or (includeSubPaths and self.edges[(s,t)][0]>0):
-                row.append(self.nodes.index(t))
-                col.append(self.nodes.index(s))
-                if includeSubPaths:
-                    count = self.edges[(s,t)].sum()
+        
+        edges_length= len(self.edges)
+
+        n_chunks = n_cores
```

Comment:
this will raise an error, since `n_cores` is not defined in this scope, I guess you wanted to write `n_chunks = self.cores`

Reply:
yes you're right, thank you